projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -216,6 +216,7 @@ protected function __construct() {
 				'query_var'             => false,
 				'capability_type'       => 'page',
 				'show_in_rest'          => true,
+				'hierarchical'          => true,
```

Comment:
@ntsekouras Can you explain this change? Why do we need the post type to be hierarchical?

Reply:
When `feedback` entities are created the post/source that they come from is saved in `post_parent` attribute of the `feedback`. This feels a bit weird to start with, but since it's been there for a long time, it seemed fine to me to update the post type to `hierarchical` and utilize this field through the REST API for filtering etc..

It doesn't create any UI issues, because the creation of `feedback` entities never happens through a UI (and it shouldn't) and we don't allow the editing of responses in the wp-admin view, which will also be removed soon nevertheless.